### DISCUSSION TITLE
Nuxt 3 server in production very slow

### DISCUSSION THREAD

**Discussion by peacheniya** [nuxt/nuxt#17305] [Questions]
hi, I ran nuxt 3 build on server and noticed that the app opens very slowly

$ yarn build
$ node .output/server/index.mjs

![image](https://user-images.githubusercontent.com/99529879/178123431-7d60f9ab-0e50-4c34-a2ac-4f12dca68671.png)

App fetches large .mjs file. This file is not even Minified,  Could you tell me how make it smaller or faster?

**Comment by Flanker72** [comment]
Seems you have a lot of dependencies bundled. Check that you haven't included, for example, FontAwesome. If you use `import "@fortawesome/fontawesome-free"` in your code - all this will be bundled.
You may change the way of how some dependencies are included (ex.: use css links instead of direct import like in example above) or create manual chunks using Vite config.
